api.md

```diff
@@ -779,7 +779,237 @@ Get all chunks older than 3 months and newer than 4 months:
+worker processes managed by Timescale. Each policy is assigned a scheduled job
+which will be run in the background to enforce it. 
+
+
+
+## add_drop_chunks_policy() [](add_drop_chunks_policy)
+Create a policy to drop chunks older than a given interval of a particular hypertable on a schedule in the background. (See drop_chunks). This implements a data retention policy and will remove data on a schedule. Only one drop_chunks policy may exists per hypertable. 
+
+#### Required Arguments [](add_drop_chunks_policy-required-arguments)
+
+|Name|Description|
+|---|---|
+| `hypertable` | (REGCLASS) Name of the hypertable to create the policy for. |
+| `older_than` | (INTERVAL) Chunks fully older than this interval when the policy is run will be dropped|
+
+#### Optional Arguments [](add_drop_chunks_policy-optional-arguments)
+
+|Name|Description|
+|---|---|
+| `cascade` | (BOOLEAN) Set to true to drop objects dependent upon chunks being dropped. Defaults to false.|
+| `if_not_exists` | BOOLEAN  Set to true to avoid throwing an error if the drop_chunks_policy already exists. A notice is issued instead. Defaults to false. |
+
+#### Returns [](add_drop_chunks_policy-returns)
+
+|Column|Description|
+|---|---|
+|`job_id`| INTEGER | Timescaledb background job id created to implement this policy|
+
+
+#### Sample Usage [](add_drop_chunks_policy-examples)
+
+
+```sql
+SELECT add_drop_chunks_policy('conditions', INTERVAL '6 months');
+```
+
+creates a data retention policy to discard chunks greater than 6 months old.
+
+
+## remove_drop_chunks_policy() [](remove_drop_chunks_policy)
+Remove a policy to drop chunks of a particular hypertable.
+
+#### Required Arguments [](remove_drop_chunks_policy-required-arguments)
+
+|Name|Description|
+|---|---|
+| `hypertable` | (REGCLASS) Name of the hypertable to create the policy for. |
+
+
+#### Optional Arguments [](remove_drop_chunks_policy-optional-arguments)
+
+|Name|Description|
+|---|---|
+| `if_exists` | (BOOLEAN)  Set to true to avoid throwing an error if the drop_chunks_policy does not exist. Defaults to false.|
+
+
+#### Sample Usage [](remove_drop_chunks_policy-examples)
+
+
+```sql
+SELECT remove_drop_chunks_policy('conditions');
+```
+
+removes the existing data retention policy for the `conditions` table.
+
+
+---
+## add_reorder_policy() [](add_reorder_policy)
+Create a policy to drop chunks older than a given interval of a particular
+hypertable on a schedule in the background. (See reorder_chunk). Only one
+reorder policy may exist per hypertable. Only chunks that are the 3rd from the
+most recent will be reordered to avoid reordering chunks that are still being
+inserted into. Once a chunk has been reordered by the background worker it will
+not be reordered again. So if one were to insert significant amounts of data in
+to older chunks that have already been reordered, it might be necessary to
+manually re-run the `reorder_chunk` function on older chunks. 
```

Comment:
Can one drop the policy and reinstate it?

Reply:
yes, but not recommended as then it will be re-run on all chunks from the beginning rather than just a few more recent ones. Eventually we'll probably have some better statistics we can use to determine when to do this.